commands/build.go

```diff
@@ -633,12 +628,18 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions, debugConfig *debug.D
 		cobrautil.MarkFlagsExperimental(flags, "root", "detach", "server-config")
 	}
 
+	flags.StringVar(&options.printFunc, "call", "", "Call a specific frontend method")
```

Comment:
As this is a new flag, we should have some documentation with basic use cases in [`Examples`](https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#examples) before having a dedicated page for it on docs website (probably in "Building images" section, WDYT @dvdksn?):

![image](https://github.com/docker/buildx/assets/1951866/5e4074ae-ef37-44f8-ba29-465c7f33cec9)

Reply:
Needs to also align tests like changing `testBuildPrint` to `testBuildCall` in https://github.com/docker/buildx/blob/7427adb9b0e78f7d86a73acc6d2179f2119d726e/tests/build.go#L800 and using this new flag.

Reply:
I'm concerned about the name of this flag (and the description of it). I don't expect most users to know what a frontend method is. Buildx should have an opinionated UX, but this seems like the opposite, where it's just a very shallow layer on top of buildkit's frontends.

Reply:
@colinhemmings  - can we get a better description for `call`? plz

Reply:
I agree that the user won't know what frontend is. This flag is for setting the build method use. So I would recommend the following help  `Set method for evaluating build ("check", "outline", "targets")`.
@dvdksn thoughts?

Reply:
Updated with the suggested description @colinhemmings 🫡

Reply:
The description is better. I still have reservations about the flag name. To me, a "call" flag doesn't really imply that this changes the build request type from the default "build an artifact from source" to something else. Neither did the `--print` flag really (maybe if you squinted hard enough), but that one was experimental. Another flag name that's more descriptive might be:

```
--mode string          Set method for evaluating build (default "build")
```

Even with a different flag name, having this switch on the `build` command is a bit misleading because the user is no longer building anything. Subrequests (as far as my understanding goes) is meant for other tasks, like analyzing source files, "introspection", or some custom output generation.

Reply:
default "build" was added